en/micro-integrator/docs/references/connectors/servicenow-connector/settingup-servicenow-instance.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the ServiceNow Instance
+
+The ServiceNow connector allows you to access the ServiceNow REST API of through WSO2 EI. To use the REST API, we need to have a [Servicenow](https://www.servicenow.com/) account. 
+
+## Signing Up for Servicenow
+
+1. Visit [Servicenow](https://www.servicenow.com/) site and sign up for an account and do the verification step. 
+  <img src="/assets/img/connectors/servicenow-1.png" title="Creating Servicenow ID" width="800" alt="Creating Servicenow ID"/>
```

Comment:
There are too many screenshots in this sequence of steps. Let's remove all non-essential ones, especially those describing navigation and UI.

Reply:
Removed